packages/devextreme/testing/testcafe/helpers/accessibilityUtils.ts

```diff
@@ -1,14 +1,20 @@
 /* eslint-disable spellcheck/spell-checker */
 import { axeCheck, createReport } from '@testcafe-community/axe';
 
-export const a11yCheck = async (t: TestController, rules: object = {}): Promise<void> => {
-  const { error, results } = await axeCheck(t, undefined, {
-    rules: {
-      'color-contrast': { enabled: false },
-      'th-has-data-cells': { enabled: false },
-      ...rules,
-    },
-  });
+const defaultOptions = {
+  rules: {
+    'color-contrast': { enabled: false },
+    'th-has-data-cells': { enabled: false },
+  },
+};
+
+export const a11yCheck = async (
+  t: TestController,
+  selector: any = undefined,
+  options: any = defaultOptions,
+):
+Promise<void> => {
```

Comment:
Let's add more strict types here and make the optional ```selector``` parameter the last one.
Plus add to the ```selector``` default value.
Something like this:

```suggestion
type A11yCheckOptions: {
  rules: Record<string, { enabled: boolean }>
}

export const a11yCheck = async (
  t: TestController,
  options: A11yCheckOptions = defaultOptions,
  selector: ElementContext | undefined = undefined,
):
Promise<void> => {
```